numbersofinterest/NumbersOfInterest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+class NumbersOfInterest
+{
+    public static function calculate($input)
+    {
+        $lines = explode(PHP_EOL, $input);
+        $numberOfSeries = array_shift($lines);
+
+        if ($numberOfSeries < 1) {
+            throw new InvalidArgumentException(
+                'Number of series to be performed "' . $numberOfSeries . '" not valid (>=1)'
+            );
+        }
+        $series = [];
+
+        for ($counter=0; $counter< $numberOfSeries; $counter++) {
+            $series[$counter] = self::calculateSeriesSequence($lines[$counter]);
+        }
+
+        return $series;
+    }
+
+    private static function calculateSeriesSequence($series)
+    {
+        list($x, $y, $n) = explode(' ', $series);
+
+        // first part of the series 
+        $currentSeries = array_fill(0, $x, $y);
+
+        for ($cursor=0; $cursor  < ($n-$x); $cursor++) {
+            $sum = 0;
+            for ($z=0; $z<$x; $z++) {
```

Comment:
would try to extract the second for into a private method, investigate what parameters could be passed